test/framework/src/main/java/org/elasticsearch/test/CorruptionUtils.java

```diff
@@ -78,8 +78,19 @@ public static void corruptFile(Random random, Path... files) throws IOException
                 checksumBeforeCorruption = CodecUtil.retrieveChecksum(input);
             }
             try (FileChannel raf = FileChannel.open(fileToCorrupt, StandardOpenOption.READ, StandardOpenOption.WRITE)) {
+                long maxPosition = raf.size();
+
+                if (fileToCorrupt.getFileName().toString().endsWith(".cfs") && maxPosition > Integer.BYTES) {
+                    // TODO: it is known that Lucene does not check the tail of CFS file (CompoundFileS, like an archive), see note at
+                    // https://github.com/apache/lucene-solr/blob/branch_7_5/lucene/core/src/java/org/apache/lucene/codecs/lucene50/
```

Comment:
Could you make this link into the following so that it doesn't rot as the code changes:

https://github.com/apache/lucene-solr/blob/60e8ee717d9cd8d2425c100c21168d6a43b78516/lucene/core/src/java/org/apache/lucene/codecs/lucene50/Lucene50CompoundReader.java#L82-L85

In fact it might be better to just put this link in the body of the first post of this PR, and make the code comment be a link to this PR, as it'll be much shorter.

Reply:
updated link in PR description (it was pointed to a single line)

the problem with the link that it violates checkstyle 140 chars max per line, will add link to this PR as it way shorter 💯